ryot/src/bevy_ryot/perspective/tests/view_point_test.rs

```diff
@@ -0,0 +1,312 @@
+use crate::bevy_ryot::perspective::*;
```

Comment:
did you consider using property testing here too? Similar to what we have in the positon module?

Reply:
Added a few relevant ones, good call.